packages/hooks/src/useClickOutside.ts

```diff
@@ -5,25 +5,33 @@ import { useUpdatedRef } from './useUpdatedRef';
 export type Handler = (event: TouchEvent | MouseEvent) => void;
 
 /**
- * Execute a handler whenever a click is registered outside of an element
+ * Execute a handler whenever a click is registered outside of a set of elements.
  */
-export function useClickOutside(ref: React.RefObject<HTMLElement>, handler: Handler) {
+export function useClickOutside<T extends HTMLElement = any>(
+  handler: Handler,
+  nodes?: (React.RefObject<any> | undefined)[],
```

Comment:
In the case of `Popover`, we need to listen to event handlers on both the popper trigger and the popover itself to make sure that clicking the trigger does not count as an "outside" click. Supporting the ability to register the event handler on multiple refs.